tests/System.Text.Primitives.Tests/PrimitiveParserIntegerTests.cs

```diff
@@ -161,6 +161,11 @@ public unsafe void ParseByteDec(string text, bool expectSuccess, byte expectedVa
             Assert.Equal(expectedValue, parsedValue);
             Assert.Equal(expectedConsumed, consumed);
 
+			result = PrimitiveParser.TryParseByte(utf8Span, out parsedValue, out consumed);
```

Comment:
Darn! It's because I haven't changed my VS settings to turn spaces into tabs when editing .tt files.